api/v2/views/instance_history.py

```diff
@@ -36,8 +36,8 @@ class InstanceStatusHistoryViewSet(MultipleFieldLookup, AuthReadOnlyViewSet):
     """
     queryset = InstanceStatusHistory.objects.all()
     serializer_class = InstanceStatusHistorySerializer
-    ordering = ('-start_date', 'instance__id')
-    ordering_fields = ('start_date', 'instance__id')
+    ordering = ('-instance__start_date', 'instance__id')
+    ordering_fields = ('-instance__start_date', 'instance__id')
```

Comment:
I *think* you can leave `start_date` in the list for `ordering_fields` and still get what you want here...
Additionally, if one wanted to sort history by start date (on a given instance, for example) they should be able to accomplish that via `?ordering=start_date`